core/src/main/scala/io/finch/ToService.scala

```diff
@@ -35,7 +35,17 @@ trait ToService[ES <: HList, CTS <: HList] {
   def apply(
     endpoints: ES,
     includeDateHeader: Boolean,
-    includeServerHeader: Boolean
+    includeServerHeader: Boolean,
+    enableMethodNotAllowed: Boolean = false
+  ): Service[Request, Response] =
+    run(endpoints, includeDateHeader, includeServerHeader, enableMethodNotAllowed, methodNotAllowed = false)
+
+  protected def run(
+    endpoints: ES,
+    includeDateHeader: Boolean,
+    includeServerHeader: Boolean,
+    enableMethodNotAllowed: Boolean,
+    methodNotAllowed: Boolean
```

Comment:
I'd probably still vote to rename it to `matchedWithoutMethod` (or something like this).